packages/polymath-ui/src/components/inputs/NumberInput/index.js

```diff
@@ -118,8 +118,9 @@ export class NumberInput extends Component<Props, State> {
     { min, max, useBigNumbers, maxDecimals }: Props
   ) {
     let value = v;
+    console.log(typeof value);
     if (typeof value === 'string') {
-      value = value.replace(/,/g, '');
+      value = value.replace(/[^0-9]+/g, '');
```

Comment:
What happens with `.` characters for decimals? Wouldn't they get replaced too? Btw this component is about 80% covered by tests I believe, so it might help to `yarn test --watch` them in case you break something without knowing

Reply:
Thanks for pointing that out. I forgot that decimals are allowed now on those fields. I've resolved and tested  with the latest commit